magento/pwa-studio
# Add newsletter in footer (#3262)

@shikhamis11 's work on adding newsletter to the footer. Merge was delayed to avoid v12 release due to missing backend support of "enabled" state flag. Originally approved: https://github.com/magento/pwa-studio/pull/3262



## Description

Add Newsletter in the footer
## Related Issue







Closes #3264

## Acceptance





### Verification Stakeholders



### Specification




## Verification Steps






#### Test scenario(s) for direct fix/feature

- Enter invalid email and submit
- Should receive an error message
- Enter valid email and submit
- Should see success toast


## Screenshots / Screen Captures (if appropriate)
![image-1-732x388](https://user-images.githubusercontent.com/25526037/125311386-ff986b80-e350-11eb-8506-14ae0505c25d.png)
![image jpeg-300x145](https://user-images.githubusercontent.com/25526037/125311402-032bf280-e351-11eb-92a1-7046519a6423.png)


## Breaking Changes (if any)





## Checklist



-   I have added tests to cover my changes, if necessary.
-   I have added translations for new strings, if necessary.
-   I have updated the documentation accordingly, if necessary.


### Resolved issues:
1. [x] resolves magento/pwa-studio#3264: Add newsletter in footer
